openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -698,7 +698,7 @@ The Credential Endpoint issues a Credential as approved by the End-User upon pre
 
 Communication with the Credential Endpoint MUST utilize TLS. 
 
-The Client can request issuance of a Credential of a certain type multiple times, e.g., to associate the Credential with different public keys/Decentralized Identifiers (DIDs) or to refresh a certain Credential.
+The Client can request the issuance of a Credential of a certain type multiple times, for example, to associate the Credential with different public keys or Decentralized Identifiers (DIDs), or to refresh a certain Credential. However, it is important to note that the ability to do so depends on whether the Issuer supports the multiple issuance of the same Credential Type or the refresh of the Credential.
 
```

Comment:
given @bc-pi 's feedback I'd suggest for now to split this PR in two different ones, where c_nonce and the clarification about multiple issuance and refresh are handled separately to prevent that our concerns about c_nonce would block an editorial clarification